zrml/prediction-markets/src/weights.rs

```diff
@@ -124,8 +124,12 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     // Storage: PredictionMarkets MarketIdsPerOpenTimeFrame (r:1 w:1)
     // Storage: PredictionMarkets MarketIdsPerCloseTimeFrame (r:1 w:1)
     // Storage: MarketCommons MarketPool (r:1 w:0)
-    fn admin_move_market_to_closed() -> Weight {
-        (79_390_000 as Weight)
+    fn admin_move_market_to_closed(o: u32, c: u32) -> Weight {
+        (35_355_000 as Weight)
+            // Standard Error: 0
+            .saturating_add((21_000 as Weight).saturating_mul(o as Weight))
+            // Standard Error: 0
+            .saturating_add((11_000 as Weight).saturating_mul(c as Weight))
```

Comment:
Good question. I think that has something to do with the storage overlay cache. There might be an small optimization for already visited paths, since `MarketIdsPerOpenTimeFrame` and `MarketIdsPerCloseTimeFrame` are closely related and the first queried element `o` (`MarketIdsPerOpenTimeFrame`) is accessed before the other one (more time to query). But I am not 100% sure, if this is correct. 

It could also be, because the node of `MarketIdsPerOpenTimeFrame` is further in the tree structure, than the other. So you know, it takes O(log(n)) to query one storage item, but this is only the worst case. Seems like `MarketIdsPerOpenTimeFrame` was more in a bad position here? But this answer can also be wrong, because storage reads and write times are taken out of the equation, because they are calculated separately.

But good question again!